src/main/java/org/cactoos/iterable/Solid.java

```diff
@@ -49,13 +47,7 @@ public Solid(final X... src) {
      * @param iterable The iterable
      */
     public Solid(final Iterable<X> iterable) {
-        super(
-            new NoNulls<>(
-                new org.cactoos.scalar.Solid<>(
```

Comment:
@victornoel why remove `NoNulls` and `Solid` here?

Reply:
@scristalli because it was useless, the tests passes without them: I think the `NoNulls` is a remnant of the past and that the `Solid` was only needed because `IterableEnvelope` was taking a `Scalar` before.